packages/tools/pluggable-widgets-tools/configs/webpack.config.common.js

```diff
@@ -52,7 +52,7 @@ const widgetConfig = {
         new CopyWebpackPlugin({
             patterns: [
                 {
-                    from: `${variables.projectPath}/src/**/*.xml`,
+                    from: posix.join(resolve(variables.projectPath, "src").replace(/\\/g, "/"), "**/*.xml"),
```

Comment:
- Why bother with `posix` since you are replacing slashes anyway?
- Are you sure you need `resolve` here?

Same apply to other places.

Reply:
Done